<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>364</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Интерактивная ручка</string>
  </property>
  <property name="windowIcon">
   <iconset resource="MainResource.qrc">
    <normaloff>:/main/webcam.png</normaloff>:/main/webcam.png</iconset>
  </property>
  <property name="toolTip">
   <string>Интерактивная ручка</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="labelMainImage">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="MainResource.qrc">:/main/hourglass.png</pixmap>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="ButtonCalibrate">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>3</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Кнопка выполнения калибровки</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Калибровка</string>
        </property>
        <property name="icon">
         <iconset resource="MainResource.qrc">
          <normaloff>:/main/calibrate.png</normaloff>:/main/calibrate.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>96</width>
          <height>96</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="ButtonSettings">
          <property name="toolTip">
           <string>Настройки программы</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MainResource.qrc">
            <normaloff>:/main/cogwheel.png</normaloff>:/main/cogwheel.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ButtonToTray">
          <property name="toolTip">
           <string>Свернуть программу в трей</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="MainResource.qrc">
            <normaloff>:/main/totray.png</normaloff>:/main/totray.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="MainResource.qrc"/>
 </resources>
 <connections/>
</ui>
